src/Sulu/Bundle/CategoryBundle/Api/Category.php

```diff
@@ -348,6 +356,17 @@ public function getTranslation()
                 return $translation;
             }
         }
+
+        return;
+    }
+
+    /**
+     * Creates a new Translation for category.
+     *
+     * @return CategoryTranslation
+     */
+    public function createTranslation()
```

Comment:
Was it really used? I don't see any advantage in having this method public, since you still have to call `setName` afterwards, otherwise the insert will still fail. Then the UPGRADE file would simply change from change this line to omit this line. Don't think that this is a bigger BC break than the current one.